tests/xua_unit_tests/src/xua_unit_test_helper.xc

```diff
@@ -26,3 +29,87 @@ void AudioHwInit()
 {
     ; // nothing
 }
+
+unsigned random(unsigned &x){
```

Comment:
using lib_random might be nicer.

Reply:
Hmm lib_random not cmake xcommon compatible yet..